doc/pki.md

```diff
@@ -0,0 +1,34 @@
+# UK Proxy Node PKI
+
+The eIDAS Proxy Node Public Key Infrastructure (PKI) is detailed in the diagram and key below.
+This PKI allows:
+
+* encryption and decryption of the eIDAS SAML request and response
+* signing and signature validation of the eIDAS SAML request and response
+* publishing of PKI metadata between the UK Proxy Node and an EU connecting country
+
+The diagram and key describe a sequential flow of of an eIDAS verify journey.
+
+![](images/proxy_node_pki.svg)
+
+Key:
+
+| Step      | Description |
+| :----:       |    :---     |
+| a   |`metadata-controller` provisions a non-extractable signing keypair in a `CloudHSM` and publishes the public key to `metadata`. The published metadata is signed with a second non-extractable key whose public certificate can be shared with the EU country for the purpose of verifying metadata signatures.
+| b   |`metadata-controller` distributes credentials to make use of the private signing key to the `translator`
+| c   | `eidas-saml-parser` stores a copy of the EU country's public encryption certificate from the EU country's metadata.
+| 1   | The EU Country produces a signed eIDAS SAML Request, and sends to `gateway` via the browser.
+| 2   | `gateway` sends the signed eIDAS SAML Request to `eidas-saml-parser` for validation and parsing.
+| 3   | `eidas-saml-parser` validates the signature of the SAML request using the public signing certificate obtained from EU country metadata.
+| 4   | `gateway` calls `verify-service-provider` to produce a signed SAML Request for Hub.
```

Comment:
What private key does the VSP use to sign the SAML request for the hub?

Reply:
I'd decided not to include the PKI between the Hub and Proxy Node, as I thought this could be explained through VSP documentation.
But am now considering including this side of the PKI, to provide the full picture.
Will talk to team about this.